osquery/tables/networking/windows/process_open_sockets.cpp

```diff
@@ -0,0 +1,270 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/core.h>
+#include <osquery/core/conversions.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "win_sockets.h"
+
+namespace osquery {
+namespace tables {
+
+WinSockets::WinSockets() : status_(Status(0, "OK")) {
+  auto pSockTable = allocateSocketTable(IPPROTO_TCP, AF_INET);
+  if (status_.ok()) {
+    tcpTable_ = static_cast<MIB_TCPTABLE_OWNER_PID*>(pSockTable);
+  } else {
+    TLOG << "Error allocating the TCP IPv4 Socket Table.";
+    return;
+  }
+
+  pSockTable = allocateSocketTable(IPPROTO_TCP, AF_INET6);
+  if (status_.ok()) {
+    tcp6Table_ = static_cast<MIB_TCP6TABLE_OWNER_PID*>(pSockTable);
+  } else {
+    TLOG << "Error allocating the TCP IPv6 Socket Table.";
```

Comment:
Also, there's inconsistent capitalization for `Socket Table` within the log lines and status messages.